feature/selector.py

```diff
@@ -597,18 +675,18 @@ def calculate_statistics(scores: pd.DataFrame,
     scores_df = scores[columns].copy()
     selected_df = selected[columns].copy()
 
+    # Group by feature
```

Comment:
can we add, group by feature "for CV results" so that this operation is explicit about CV

Reply:
Updated in this [commit](https://github.com/fidelity/selective/pull/3/commits/65c53462b58b6aa13e3670d5e3fe6fef00abf5b4).